src/main/scala/com/fulcrumgenomics/vcf/DownsampleVcf.scala

```diff
@@ -99,7 +98,7 @@ object DownsampleVcf extends LazyLogging {
   def downsampleAndRegenotype(gt: Genotype, proportion: Double, random: Random, epsilon: Double): Genotype = {
     val oldAds = gt.getOrElse[IndexedSeq[Int]]("AD", throw new Exception(s"AD tag not found for sample ${gt.sample}"))
     val newAds = downsampleADs(oldAds, proportion, random)
-    val likelihoods = Likelihoods(newAds)
+    val likelihoods = Likelihoods(newAds, epsilon = epsilon)
```

Comment:
This was a "bug" whereby `epsilon` was not passed down the call stack so only the default was being used.